### ISSUE TITLE
Processing error. Bit_rate

### ISSUE THREAD

**Issue by OzGav** [music-assistant/hass-music-assistant#599]
### What version of Music Assistant has the issue?

2022.7.0b0

### The problem

Had errors on start as per other issue but MA did appear to start. During what appear to be the initial scan I saw the error below in the log.

### How to reproduce

Initial scan

### Relevant log output

```
Logger: music_assistant.music.file
Source: /usr/local/lib/python3.9/site-packages/music_assistant/music_providers/filesystem.py:199 
First occurred: 8:59:36 AM (5 occurrences) 
Last logged: 8:59:40 AM

Error processing /media/nasMusic/Coldplay/Live 2003/04. Coldplay - Daylight (Live 2003).flac - 'bit_rate'
Error processing /media/nasMusic/Coldplay/Live 2003/13. Coldplay - The Scientist (Live 2003).flac - 'bit_rate'
Error processing /media/nasMusic/Coldplay/Live 2003/17. Coldplay - Life is for Living (Live 2003).flac - 'bit_rate'
Error processing /media/nasMusic/Coldplay/Live 2003/05. Coldplay - Trouble (Live 2003).flac - 'bit_rate'
Error processing /media/nasMusic/Coldplay/Live 2003/07. Coldplay - Dont Panic (Live 2003).flac - 'bit_rate'
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/music_assistant/music_providers/filesystem.py", line 190, in sync_library
    if track := await self._parse_track(entry.path):
  File "/usr/local/lib/python3.9/site-packages/music_assistant/music_providers/filesystem.py", line 481, in _parse_track
    tags = await parse_tags(track_path)
  File "/usr/local/lib/python3.9/site-packages/music_assistant/helpers/tags.py", line 199, in parse_tags
    return AudioTags.parse(json.loads(res))
  File "/usr/local/lib/python3.9/site-packages/music_assistant/helpers/tags.py", line 165, in parse
    bit_rate=int(raw["format"]["bit_rate"]),
KeyError: 'bit_rate'

```

### What version of Home Assistant Core are your running

2022.6.6

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Raspberry Pi

**Comment by erkr** [comment]
Latest beta was for core-2022.7.x only. Please check if the issue still persists when you upgraded core

**Comment by marcelveldt** [comment]
Fixed in 2022.7.0 beta 1

**Comment by ovizii** [comment]
> Fixed in 2022.7.0 beta 1

Do you think that will also fix this error or should I open a new issue for it?

```
2022-07-08 10:49:44 ERROR (MainThread) [music_assistant.music.file] Error processing /media/nas-music/Fettes Brot/Demotape/14 - Konnten Sie mich kurz küssen_.mp3 - 'tags'
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/music_assistant/music_providers/filesystem.py", line 189, in sync_library
    if track := await self._parse_track(entry.path):
  File "/usr/local/lib/python3.10/site-packages/music_assistant/music_providers/filesystem.py", line 480, in _parse_track
    tags = await parse_tags(track_path)
  File "/usr/local/lib/python3.10/site-packages/music_assistant/helpers/tags.py", line 199, in parse_tags
    return AudioTags.parse(json.loads(res))
  File "/usr/local/lib/python3.10/site-packages/music_assistant/helpers/tags.py", line 151, in parse
    for key, value in raw["format"]["tags"].items()
KeyError: 'tags'
```

**Comment by marcelveldt** [comment]
@ovizii I will fix that for the next beta (2)
Hmmm that is actually an error indicating that the file does not have any tags at all, is that intentional ?
